### PR TITLE
Add `-loader` suffix in webpack, closes #2410

### LINKED ISSUE #2410: Javascript bundle version is broken in 0.19.0
The 0.19.0 build is only a few kilobytes, instead of a couple of megabytes.

image

And when I run it I get:

```sh
$ node ./yarn-0.19.0.js
~/Downloads/yarn-0.19.0.js:74
(function webpackMissingModule() { throw new Error("Cannot find module \"/home/ubuntu/yarn/src/cli/index.js\""); }());
                                   ^

Error: Cannot find module "/home/ubuntu/yarn/src/cli/index.js"
    at webpackMissingModule (~/Downloads/yarn-0.19.0.js:74:42)
    at Object.<anonymous> (~/Downloads/yarn-0.19.0.js:74:115)
    at __webpack_require__ (~/Downloads/yarn-0.19.0.js:21:30)
    at ~/Downloads/yarn-0.19.0.js:67:18
    at Object.<anonymous> (~/Downloads/yarn-0.19.0.js:70:10)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)

```

Node.js: v7.1.0
OS: Ubuntu 16.04

### PR DESCRIPTION
**Pull request by QWp6t**
Bundled builds were not being built correctly..

Example: https://github.com/yarnpkg/yarn/releases/download/v0.19.0/yarn-0.19.0.js
Issue: https://github.com/yarnpkg/yarn/issues/2410

## tl;dr

Just add `-loader` suffix to loaders in to scripts/build-webpack.js. ezpz

## Details

Previously, packages named with `-loader` suffix could be specified as loaders within Webpack without requiring the suffix to be named explicitly. Webpack 2.1.0-beta.26 no longer looks for `-loader` suffix automatically, and packages have to therefore be named explicitly or `-loader` can be added to `resolveLoader.moduleExtensions`.

See: https://github.com/webpack/webpack/releases/tag/v2.1.0-beta.26

This PR explicitly adds `-loader` suffix, and now bundles are correctly built.

### Before

```sh
~/yarn-0.19.0  (yarn@0.19.0)
λ node scripts/build-webpack.js
[]

~/yarn-0.19.0  (yarn@0.19.0)
λ
```

### After

```sh
~/yarn-0.19.0  (yarn@0.19.0)
λ node scripts/build-webpack.js
[ 'node_modules/ansi-escapes/index.js',
  'node_modules/ansi-regex/index.js',
  'node_modules/ansi-styles/index.js',
  'node_modules/array-find-index/index.js',
  'node_modules/asn1/lib/ber/errors.js',
  # ... etc ...
  'src/util/promise.js',
  'src/util/request-manager.js',
  'src/util/stream.js',
  'src/util/version.js' ]

~/yarn-0.19.0  (yarn@0.19.0)
λ
```

### CODE REVIEW

**Comment by bestander** [comment]
Thanks, cherry-picking to 0.19.
0.19.1 will be released next week
